internal/catalogmetadata/filter/successors.go

```diff
@@ -20,17 +20,17 @@ func SuccessorsOf(installedBundle *ocv1alpha1.BundleMetadata, channels ...declcf
 
 	installedBundleVersion, err := mmsemver.NewVersion(installedBundle.Version)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("parsing installed bundle version %q: %w | installedBundle %v", installedBundle.Version, err, installedBundle)
```

Comment:
```suggestion
		return nil, fmt.Errorf("parsing installed bundle %q version %q: %w | installedBundle %v", installedBundle.Name, installedBundle.Version, err)
```

Reply:
Ah shoot, some of that is still debugging changes I didn't rollback - thanks for catching that!

Reply:
should be updated in https://github.com/operator-framework/operator-controller/pull/1068/commits/8c127588ceff671306f53a290b9f4acec2247f75